snomed/com.b2international.snowowl.snomed.core.rest/src/com/b2international/snowowl/snomed/core/rest/SnomedRf2ExportRestService.java

```diff
@@ -79,7 +80,7 @@
 			.setIncludePreReleaseContent(params.isIncludeUnpublished())
 			.setModules(params.getModuleIds())
 			.setRefSets(params.getRefSetIds())
-			.setCountryNamespaceElement(params.getNamespaceId())
+			.setCountryNamespaceElement(getCountryNamespaceElement(params))
```

Comment:
What happened with the `namespaceId` argument? It's no longer being used, which is probably not what we want.